src/config.js

```diff
@@ -0,0 +1,3 @@
+export const serverBaseUrl = process.env.NODE_ENV === 'production'
```

Comment:
Good stuff 👍

Reply:
An improvement would be to allow for overwriting the URL.

So let's say f.x. I've got the development app running, but I want it to talk to the remote backend (to get more data, or negate having to run docker on my machine, or proper testing of HTTP)

I'm imagining something like `yarn run start API_HOST='api-staging.veritus.io'`

Reply:
What remote backend would you be talking to except prod?

Reply:
Staging

Which is non-existent right now I grant you

Reply:
But when running the App in development mode, it's always pointing towards local host.
This is something I see myself changing quite often, where I don't want to go into the file and manually change it.

Instead I'd like to give the option via run commands

Reply:
Something about this [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables) that I can take a look at later.